src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -178,6 +178,25 @@ public void removeParticipation(Participation key) {
     }
 
 
+    /**
+     * Removes a tutorial, denoted by {@code key} from this {@code AddressBook}.
+     * {@code key} must exist in the address book.
+     */
+    public void closeTutorial(Tutorial key) {
```

Comment:
I know this method name is aligned with the command name, but will it be better if we name it 'removeTutorial' to align it with the rest of the AddressBook methods such as removePerson and removeParticipation?

Reply:
I was thinking that this might be better as well, more intuitive for future developments too